voila/execute.py

```diff
@@ -109,7 +109,7 @@ def strip_code_cell_errors(self, cell):
 
         error_outputs = [output for output in outputs if output['output_type'] == 'error']
 
-        error_message = 'There was an error when executing cell [{}]. {}'.format(cell['execution_count'], self.cell_error_instruction)
+        error_message = 'There was an error when executing cell [{cell["execution_count"]}]. {self.cell_error_instruction}'
```

Comment:
Looks like there is missing 'f'?

```suggestion
        error_message = f'There was an error when executing cell [{cell["execution_count"]}]. {self.cell_error_instruction}'
```